test/api/exclude-members.test.js

```diff
@@ -25,6 +25,10 @@ test('tribes.excludeMembers', async t => {
     )
 
     await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name })
+    await p(setTimeout)(100)
+
+    const newPersonGotGroup = await p(newPerson.tribes.get)(groupId)
+    t.false(!!newPersonGotGroup.excluded, 'new person is not excluded yet')
```

Comment:
this is confusing to read to me, and `!!` hides when something came back as e.g. `undefined` (coerces it to `false`)
Can you just do:
```suggestion
    t.equal(newPersonGotGroup.excluded, false, 'new person is not excluded yet')
```

Reply:
wait oops it's supposed to be undefined. but i'll test for that instead